ssv/spectest/run_test.go

```diff
@@ -229,6 +229,10 @@ func fixRunnerForRun(t *testing.T, runnerMap map[string]interface{}, ks *testing
 		ret.(*ssv.ProposerRunner).ProducesBlindedBlocks = blindedBlocks.(bool)
 	}
 
+	if gasLimit, ok := runnerMap["GasLimit"]; ok {
+		ret.(*ssv.ValidatorRegistrationRunner).GasLimit = uint64(gasLimit.(float64))
```

Comment:
ret will always be `ssv.ValidatorRegistrationRunner` when `runnerMap["GasLimit"]` is set?

Reply:
Yes, it's done the same way as the code above. If it's not there, it means that's JSON is from a previous version